tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -95,7 +95,7 @@ sub execute_script_run {
 
     if ($ret) {
         save_screenshot;
-        $ret =~ s/$pattern//g;
+        $ret =~ s/[\r\n]+$pattern[\r\n]+//g;
```

Comment:
That should probably go into its own commit with a good explanation of the reason of the change.